src/server/JasmineGraphInstanceService.cpp

```diff
@@ -2209,7 +2209,164 @@ void *instanceservicesession(void *dummyPt) {
+                }
+                std::cout << "Thread [A]: " << threadCount << " joined" << std::endl;
+            }
+        }
+        else if (line.compare(JasmineGraphInstanceProtocol::SHARE_BUCKETS_TO_COORDINATOR) == 0) {
+            instance_logger.log("Received : " + JasmineGraphInstanceProtocol::SHARE_BUCKETS_TO_COORDINATOR, "info");
+            write(connFd, JasmineGraphInstanceProtocol::OK.c_str(), JasmineGraphInstanceProtocol::OK.size());
+            instance_logger.log("Sent : " + JasmineGraphInstanceProtocol::OK, "info");
+            bzero(data, INSTANCE_DATA_LENGTH);
+
+            //Read coordinator host name
+            read(connFd, data, INSTANCE_DATA_LENGTH);
+            string coordinatorHost = (data);
+            coordinatorHost = utils.trim_copy(coordinatorHost, " \f\n\r\t\v");
+            instance_logger.log("Received HostName: " + coordinatorHost, "info");
+
+            //Read coordinator port from socket
+            bzero(data, INSTANCE_DATA_LENGTH);
+            read(connFd, data, INSTANCE_DATA_LENGTH);
+            string coordinatorPort = (data);
+            coordinatorPort = utils.trim_copy(coordinatorPort, " \f\n\r\t\v");
+            instance_logger.log("Received Port: " + coordinatorPort, "info");
+
+            //Read coordinator data port from socket
+            bzero(data, INSTANCE_DATA_LENGTH);
+            read(connFd, data, INSTANCE_DATA_LENGTH);
+            string coordinatorDataPort = (data);
+            coordinatorDataPort = utils.trim_copy(coordinatorDataPort, " \f\n\r\t\v");
+            instance_logger.log("Received Data Port: " + coordinatorDataPort, "info");
+
+            //Read graph ID from socket
+            bzero(data, INSTANCE_DATA_LENGTH);
+            read(connFd, data, INSTANCE_DATA_LENGTH);
+            string graphID = (data);
+            graphID = utils.trim_copy(graphID, " \f\n\r\t\v");
+            instance_logger.log("Received Graph ID: " + graphID, "info");
+
+            //Read cluster count from socket
+            bzero(data, INSTANCE_DATA_LENGTH);
+            read(connFd, data, INSTANCE_DATA_LENGTH);
+            int noClusters = stoi(data);
+            graphID = utils.trim_copy(graphID, " \f\n\r\t\v");
+            instance_logger.log("Received Graph ID: " + graphID, "info");
+
+            for (int i = 0; i < noClusters; i++) {
+                string filename = "AttrCluster_" + graphID + to_string(i) + ".txt";
+                string filepath =
+                        utils.getJasmineGraphProperty("org.jasminegraph.server.instance.entityresolutionfolder") + filename;
+                JasmineGraphInstance::sendFileThroughService(coordinatorHost, stoi(coordinatorPort), filename, filepath);
+            }
+
+        }
+        else if (line.compare(JasmineGraphInstanceProtocol::SHARE_CLUSTER_FILES) == 0) {
+            instance_logger.log("Received : " + JasmineGraphInstanceProtocol::SHARE_BUCKETS_TO_ORG, "info");
```

Comment:
`SHARE_BUCKETS_TO_ORG` shouldn't this be share cluster files?

Reply:
Changed to `SHARE_CLUSTER_FILES`

Reply:
Ok